### PR TITLE
Version 5

### PR DESCRIPTION
**Pull request by khornberg**

### CODE REVIEW

File: request_signer/validator.py [hunk 1]
```diff
@@ -56,8 +62,11 @@ def url_path(self):
     def client(self):
         if not self.signature or not self.client_id:
             return False
-
-        return models.AuthorizedClient.get_by_client(self.client_id)
+        if getattr(settings, 'API_KEYS', None):
+            Client = namedtuple('client', ['private_key'])
+            return Client(settings.API_KEYS.get(self.client_id, ''))
+        logger.error('API_KEYS not found in settings')
```
Comment (mattjmorrison) [inline request_signer/validator.py]: Rather than logging an error, should this be Django's `ImpoperlyConfigured` exception?

**Comment by khornberg** [comment]
@mattjmorrison Good idea; updated

**Comment by mattjmorrison** [comment]
Before pushing this out anywhere it might be a good idea to run this thing: https://github.com/imtapps/egg_search and make sure that everything out there is locking in the version appropriately - I'm fairly confident that it is, but if this breaks anything the world breaks

**Comment by khornberg** [comment]
Already checked. 5.0 will not be picked up by anything

**Pull request merged**